account_factoring_receivable_balance/models/subrogation_receipt.py

```diff
@@ -0,0 +1,237 @@
+# © 2022 David BEAL @ Akretion
+# © 2022 Alexis DE LATTRE @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SubrogationReceipt(models.Model):
+    _name = "subrogation.receipt"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _check_company_auto = True
+    _rec_name = "factor_type"
+    _order = "target_date DESC"
+    _description = "Customer balance data for factoring"
+
+    factor_journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        string="Journal",
+        domain="[('factor_type', '!=', False)," "('type', '=', 'bank')]",
```

Comment:
@dreispt 
In my experience (minor) but the journal associated to account.move used to send data to factor etablishment (invoices and credit notes) can be a `misc` one.
In this kind of journal you have not a lot settings, then it's less confusing to use this kind in this context.
At least, we may consider to offer a choice.
Don't you think Daniel ?

Reply:
@bealdav Good question, this was something I struggled with.
For FactoFrance we need to also report the payments on Invoices.
So I settled with using a Bank journal, allowing you to post payments on factored invoices.

Reply:
> @bealdav Good question, this was something I struggled with.
> For FactoFrance we need to also report the payments on Invoices.
> So I settled with using a Bank journal, allowing you to post payments on factored invoices.

@dreis for Factofrance I strongly recommend you look at this other module I did initially, it is used in production since 3 years in a very big company and was designed for Factofrance: https://github.com/akretion/odoo-factoring/tree/14.0/account_factoring

You can find a demo here: https://youtu.be/N7x7MtfEF1g?si=0FbU3P1O8Iv8MgzE